packages/account/package.json

```diff
@@ -55,21 +55,23 @@
     "access": "public"
   },
   "devDependencies": {
+    "@ethersproject/providers": "^5.7.2",
+    "@ethersproject/wallet": "^5.7.0",
     "@types/node": "^20.11.10",
     "esbuild": "^0.19.11",
     "esbuild-plugin-tsc": "^0.4.0",
     "gh-pages": "^6.1.1",
+    "lru-cache": "^10.0.1",
```

Comment:
It looks like we don't use this dependency anywhere, we could remove it.

Reply:
Yeah we can do in develop branch 👍